packages/dashboards/addon/templates/dashboards/dashboard/widgets/widget.hbs

```diff
@@ -1,113 +1,118 @@
-{{!-- Copyright 2018, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
 <div class='navi-report-widget'>
-    {{!-- Breadcrumb --}}
-    <div class='navi-report-widget__breadcrumb'>
-        {{#if (feature-flag 'enableDirectory')}}
-            {{#link-to 'directory.my-data' class='navi-report-widget__breadcrumb-link'}}Directory{{/link-to}}
-        {{else}}
-            {{#link-to 'dashboards' class='navi-report-widget__breadcrumb-link'}}Dashboards{{/link-to}}
-        {{/if}}
-        {{navi-icon 'angle-right'}}
-        {{#with (model-for 'dashboards.dashboard') as |parentDashboard|}}
-          {{#link-to 'dashboards.dashboard' parentDashboard.id class='navi-report-widget__breadcrumb-link'}}
-            {{parentDashboard.title}}
-          {{/link-to}}
-        {{/with}}
-        {{navi-icon 'angle-right'}}
-    </div>
+  {{!-- Breadcrumb --}}
+  <div class='navi-report-widget__breadcrumb'>
```

Comment:
Can we start using ember conventions for templates? Just give us less to clean up later.

indentation looks cool, but use double quotes.